src/bitmessagecli.py

```diff
@@ -394,53 +411,54 @@ def subscribe():
         address = userInput("What address would you like to subscribe to?")
 
         if (address == "c"):
-                usrPrompt = 1
-                print ' '
-                main()
-        elif (validAddress(address)== False):
+            usrPrompt = 1
+            print ' '
+            main()
+        elif (validAddress(address) == False):
```

Comment:
`validAddress(address) is False`

Reply:
Will be fixed with superfluous-parens issues